src/sentry/options/defaults.py

```diff
@@ -1754,6 +1754,11 @@
     default=10000,
     flags=FLAG_AUTOMATOR_MODIFIABLE,
 )
+register(
+    "on_demand.max_widget_cardinality.on_query_count",
+    default=100,
```

Comment:
```suggestion
    default=50,
```
Let's bump this down to `50` actually, I'd prefer we start low and bump it up later if needed. Widgets will fall back to indexed data anyway.